assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -230,6 +230,23 @@ function APIProvider({ children }) {
     return apiFetch({ path: addQueryArgs(users, { per_page: '-1' }) });
   }, [users]);
 
+  /**
+   * Status check, submit html string.
+   *
+   * @param {string} HTML string.
+   * @return {Promise} Result promise
+   */
+  const getStatusCheck = useCallback(
+    (content) => {
+      return apiFetch({
+        path: statusCheck,
+        data: { content: encodeMarkup ? base64Encode(content) : content },
```

Comment:
Thinking out loud: would be nice if this could be steered via a parameter, e.g. `getStatusCheck({content: content, encode: true)`

Then we can run `getStatusCheck` twice to see if one works.

Reply:
What is the point of sending it twice here?

Reply:
The idea was to know whether the first one is blocked by a firewall, and whether the second call works around the blocking.

Reply:
But yeah, probably not needed for now